extensions/measurement-tracking/src/viewports/TrackedCornerstoneViewport.js

```diff
@@ -345,9 +353,13 @@ function _getCornerstoneStack(displaySet, dataSource) {
   return stack;
 }
 
-async function _getViewportData(dataSource, displaySet) {
+async function _getViewportData(dataSource, displaySet, imageIndex) {
   let viewportData;
 
+  if (imageIndex !== undefined) {
+    displaySet.frameIndex = imageIndex;
```

Comment:
`displaySet` should also have an `imageIndex` property.
For now, it should always be set, even if it's value is `undefined`

We _might_ choose to default to `0`. Not 100% sure about that yet
This function should have `JSDoc`

Reply:
Updated.